src/data/set/finite.lean

```diff
@@ -534,6 +534,11 @@ lemma eq_of_subset_of_card_le {s t : set α} [fintype s] [fintype t]
 (eq_or_ssubset_of_subset hsub).elim id
   (λ h, absurd hcard $ not_le_of_lt $ card_lt_card h)
 
+lemma subset_iff_to_finset_subset (α : Type*) (s t : set α) [fintype s] [fintype t] :
```

Comment:
```suggestion
lemma subset_iff_to_finset_subset (s t : set α) [fintype s] [fintype t] :
```

surely there's `variables (α : Type*)` above this? Although you'll need to check if there are any typeclasses on `α` at this point in the file first.

Reply:
Sorry yeah, being super careless. It is already declared and there aren't any typeclasses. Although the above lemmas have `s` and `t` as implicit so maybe I should update that too?